readme.md

```diff
@@ -33,6 +33,14 @@ and add the private key from Firebase Admin SDK.
 - run `java -jar covid19-1.0.0-SNAPSHOT.jar`. Check if the jar version matches.
 - NOTE: access logs will get stored in the tomcat folder (tomcat/access/).
  These logs can be disabled from the application.yml
+ 
+### with Docker
+- build project using `mvn clean install`
+- unzip the `target/covid19-1.0.0-SNAPSHOT.zip` to `target/covid19-1.0.0-SNAPSHOT`
+- modify the db url in the application.yml (replace localhose with container name) as below  (FIXME: automate with spring profiles)

```

Comment:
great suggestion! are you able to make this change? do not forget to create a issue :)

Reply:
https://github.com/ICTASL/COVID-19/issues/97